src/r2mm/installing/profile_installers/GenericProfileInstaller.ts

```diff
@@ -440,7 +440,11 @@ export default class GenericProfileInstaller extends ProfileInstallerProvider {
             files: Array.from(existing.entries())
         }
         await FsProvider.instance.writeFile(path.join(profile.getPathOfProfile(), "_state", `${mod.getName()}-state.yml`), yaml.stringify(mft));
-        await ConflictManagementProvider.instance.overrideInstalledState(mod, profile);
+        const err = await ConflictManagementProvider.instance.overrideInstalledState(mod, profile);
```

Comment:
Could you verify that this doesn't cause any issues when installing BONEWORKS mods?
I've looked through the code and have a feeling it'll be fine, but would be good to sanity check.